src/js/injection_script.js

```diff
@@ -1,63 +1,13 @@
-    "shortmoz.link": "bypasses/shortmoz.js",
-    "/sourceforge.net/projects/.+/files/.+/download/": "bypasses/sourceforge.js",
-    "srt.am": "bypasses/srtam.js",
-    "sub4unlock.com": "bypasses/sub4unlock.js",
-    "syosetu.org": "bypasses/syosetu.js",
-    "tik.lat": "bypasses/tiklat.js",
-    "uiz.io": "bypasses/uiz.js",
-    "uiz.app": "bypasses/uiz.js",
-    "tlkm.id": "bypasses/uiz.js",
-    "vk.com/away.php?to=": "bypasses/vk.js",
-    "wadooo.com": "bypasses/wadooo.js",
-    "ytsubme.com": "bypasses/ytsubme.js"
-};
+const bypasses = {"acortalink.me":"bypasses/acortalink.js","www93.davisonbarker.pro":"bypasses/adfly.js","www96.lowrihouston.pro":"bypasses/adfly.js","akoam.to":"bypasses/akoam.js","anonym.ninja":"bypasses/anonym.js","brpaper.com":"bypasses/brpaper.js","cb.run":"bypasses/cbrun.js","cb.click":"bypasses/cbrun.js","complete2unlock.com":"bypasses/complete2unlock.js","cpmlink.net":"bypasses/cpmlink.js","daominhha.com/download":"bypasses/daominhha.js","earnme.club":"bypasses/earnme.js","usanewstoday.club":"bypasses/earnme.js","fastforward.team":"bypasses/fastforward.js","fc-lc.com":"bypasses/fclc.js","filedm.com":"bypasses/filedm.js","filehorse.com":"bypasses/filehorse.js","forex1pro.com":"bypasses/forex1pro.js","4shared.com":"bypasses/fourshared.js","fssquad.com":"bypasses/fssquad.js","gamesmega.net":"bypasses/gamesmega.js","gixen.com/home_1.php":"bypasses/gixen.js","leitup.com":"bypasses/leitup.js","liblink.pl":"bypasses/liblink.js","linkspy.cc":"bypasses/linkspy.js","linkvertise.com":"bypasses/linkvertise.js","linkvertise.net":"bypasses/linkvertise.js","link-to.net":"bypasses/linkvertise.js","lnk2.cc":"bypasses/lnk2cc.js","longfiles.com":"bypasses/longfiles.js","filepuma.com":"bypasses/longfiles.js","portableapps.com":"bypasses/longfiles.js","indishare.org":"bypasses/longfiles.js","solvetube.site":"bypasses/longfiles.js","mangalist.org":"bypasses/mangalist.js","mydramalist.com":"bypasses/mydramalist.js","onepieceex.net":"bypasses/onepieceex.js","online-fix.me/ext/":"bypasses/onlinefix.js","oracle.com":"bypasses/oracle.js","pirateproxy.wtf":"bypasses/pirateproxy.js","rekonise.com":"bypasses/rekonise.js","sfile.mobi":"bypasses/sfile.js","sfile.xyz":"bypasses/sfile.js","apkmos.com":"bypasses/sfile.js","links.shortenbuddy.com":"bypasses/shortenbuddy.js","shortmoz.link":"bypasses/shortmoz.js","/sourceforge.net/projects/.+/files/.+/download/":"bypasses/sourceforge.js","squidssh.com/li/go.php":"bypasses/squidssh.js","goodssh.com/li/go.php":"bypasses/squidssh.js","srt.am":"bypasses/srtam.js","sub4unlock.com":"bypasses/sub4unlock.js","syosetu.org":"bypasses/syosetu.js","tik.lat":"bypasses/tiklat.js","uiz.io":"bypasses/uiz.js","uiz.app":"bypasses/uiz.js","tlkm.id":"bypasses/uiz.js","vk.com/away.php?to=":"bypasses/vk.js","wadooo.com":"bypasses/wadooo.js","ytsubme.com":"bypasses/ytsubme.js"};
+    const script_src = new URL(document.currentScript.src)
+    const ext_base_URL = script_src.searchParams.get("ext_base_URL")
```

Comment:
this is useless, you need to add this to the generator, this file is generated

Reply:
yea that's prettier reformatting files, should I add injection script to gitignore ?https://softwareengineering.stackexchange.com/questions/391804/should-generated-documentation-be-stored-in-a-git-repository

Reply:
I get what you mean now

Reply:
Probably should, there's no need to keep it around if it changes every time build.js is ran.